src/components/legend/constants.js

```diff
@@ -12,5 +12,6 @@ module.exports = {
     scrollBarWidth: 4,
     scrollBarHeight: 20,
     scrollBarColor: '#808BA4',
-    scrollBarMargin: 4
+    scrollBarMargin: 4,
+    DBLCLICKDELAY: 300
```

Comment:
To avoid duplication we should probably put this (and a few other things) into [constants/interactions](https://github.com/plotly/plotly.js/blob/master/src/constants/interactions.js) and reference it from legend and [dragelement](https://github.com/plotly/plotly.js/blob/master/src/components/dragelement/index.js#L15) - which for obsolete reasons currently gets these constants out of [cartesian/constants](https://github.com/plotly/plotly.js/blob/master/src/plots/cartesian/constants.js)